application/Core/GuestController.php

```
@@ -28,7 +28,7 @@ public function __construct()
     {
         parent::__construct('user_type', 2);
 
-        $this->load->model('user_clients/mdl_user_clients');
+        $this->load->model('user_clients/user_client');
 
         $user_clients = $this->mdl_user_clients->assigned_to($this->session->userdata('user_id'))->get()->result();
```

PHPMD (2.15.0)

The variable $user_clients is not named in camelCase. (undefined)

(CamelCaseVariableName)